integrationtests/mayavi/common.py

```diff
@@ -341,6 +341,7 @@ def compare_image_with_saved_image(src_img, img_fname, threshold=10,
                 if alt_err < min_err:
                     # image is a better match.
                     err_index = count
+                    test_failed = 0
```

Comment:
I think this is not correct, this still is a failed image as the error is > threshold.  Instead we should checkin a new image for VTK 7.